src/components/Calculator.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import './Calculator.css';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] [OPTIONAL] Good job so far! Please fix linter errors without this kind of comment `/* eslint-disable ... */` because this type of comment is used in very few cases, and in your case, you can solve such errors yourself, and I think if implement my suggestion that this will increase your work experience and skills 😃

Reply:
Hello, @mukhammadrizooff could you kindly share how one can fix the error? Because it is forcing me to use a functional component yet the instructions require a class component. 🙏